static/app/views/monitors/utils/useMonitorDates.tsx

```diff
@@ -9,7 +9,7 @@ import type {TimeWindowConfig} from '../components/overviewTimeline/types';
 
 interface Options {
   /**
-   * The width of the timeline influences ho we caluclate the rollup value
+   * The width of the timeline influences ho we calculate the rollup value
```

Comment:
```suggestion
   * The width of the timeline influences how we calculate the rollup value
```

in case its easier to just take the gh suggestion

Reply:
lol oops